test/testscripts/candlestick_scale_color.jl

```diff
@@ -0,0 +1,13 @@
+using Gadfly, MarketData
```

Comment:
This plot could be added to the above testscript, and the 2 plots hstack'd or vstack'd.  This makes for fewer testscripts overall (what I prefer).

Reply:
:ok_hand: